content/roles_and_responsibilities.md

```diff
@@ -1,19 +1,64 @@
+
+The Quizmaster must:
+- Identify the question type of each question and begin every question with, “question number_____, question”.
+- Rule immediately upon an obviously correct or incorrect answer if there is no indication from the answer judge to the contrary. 
+    - When an answer is in question (different than what is on the card), the quizmaster will discuss with the answer judge.
+- During the 30-second answering period use the following prompts:
+    - If a quizzer has provided all the information in the answer, except the identification of a pronoun present which is part of the answer, the quizmaster will ask the quizzer to identify the specific pronoun, (example: “Can you identify ‘He’?” or “Can you identify ‘They’?”) The quizmaster is not allowed to say anything other than what the quizzer has already said when asking for the identification.   A pronoun may need to be identified only if the identification (antecedent) is in context.
+    - On a Chapter-Verse Reference Question (CVR) or Chapter-Only Reference Question (CR), if the quizzer has given all the information in both the question and the answer and the complete question was not read, the quizmaster will ask, "What is your question?"
+    - On a Finish-The-Verse (FTV), Quote, or Finish This (FT) type question, once the quizzer has quoted to the end of the required verse or verses, if the quote is not yet word-perfect, the quizmaster will say, "Again."  This will occur as often as needed until the quizzer quotes the verse perfectly or the 30-second time limit expires.  
+        - The quizmaster is not allowed to say “again” until the quizzer has come to the end of the verse, each time through, as this would give the quizzer information about where their mistake was made.
+    - Once all of the information in both the question and answer have been given, but something needs to be fixed (e.g. Unique word) the quizmaster will say “Again”.
+    - On a Situation (SIT) type question, when the quizzer has completed the Quotation the quizmaster will say, “Quote is complete.”
+    - On an INT or MA Question, if all of the information in the answer has been given, but the quizzer is missing information in the question, the quizmaster will say “More”.
+    - On a Situation question, because the quotation is part of the question, the quizmaster will say “More” when required information from the quotation is missing.
+    - On a Q, Q2, CVR, MACVR, CR, MACR question, if either the chapter or verse number were not completely finished when the quizzer jumped, the quizmaster will say “More”. 
+        - Note: On a CVR/MACVR, CR/MACR Question once the quizzer has provided all the information in both the question and answer the quizmaster will say “What is your question”, if the response to the question supplied by the quizzer is “More”, the quizzer should consider whether the quizmaster had completely pronounced the chapter or verse number. 
```

Comment:
This implies that the Reference is not considered part of the "Question" right? Since the quizzer has to have provided all the information in the Question and Answer to be prompted for their reference question?  That seems problematic to me.

Reply:
This goes to my earlier comment about "question" being overloaded.